src/module/tests/SuccessTest.ts

```diff
@@ -26,6 +26,7 @@ import { SuccessTestEffectsFlow } from '../effect/flows/SuccessTestEffectsFlow';
 import { SR5ActiveEffect } from '../effect/SR5ActiveEffect';
 import { Translation } from '../utils/strings';
 import { GmOnlyMessageContentFlow } from '../actor/flows/GmOnlyMessageContentFlow';
+import { MatrixTestData } from './MatrixTest';
```

Comment:
It shouldn't be necessary to include MatrixTest imports in the top level class. If there is a need for that, we should look into resolving that.

For `executeMessageAction` I'd rather have it use `as any` or ignore the testing issue than introducing circular imports.

Reply:
Change it to `any` for now. Maybe in the future we could make a static "getTargetDocuments" or something on the Test to use

Reply:
That area isn't well designed, yes. Let's see what further needs we have for it and restructure. It's not much code in any way.